src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,21 @@
-export const TodoInfo = () => {};
+import './TodoInfo.scss';
+
+import React from 'react';
+import { UserInfo } from '../UserInfo';
+
+import { Todo } from '../../types/types';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => (
+  <article
+    data-id={todo.id}
+    className={`TodoInfo ${todo.completed ? 'TodoInfo--completed' : ''}`}
+  >
+    <h2 className="TodoInfo__title">{todo.title}</h2>
+
+    <UserInfo user={todo.user} />
```

Comment:
and add here some check if user exists
```suggestion
    {todo.user && <UserInfo user={todo.user} />}
```